lib/vanagon/platform/windows.rb

```diff
@@ -85,7 +85,27 @@ def generate_nuget_package(project)
       # @param project [Vanagon::Project] project to name
       # @return [String] name of the windows package for this project
       def nuget_package_name(project)
-        "#{project.name}-#{@architecture}-#{project.version}.#{project.release}.nupkg"
+        "#{project.name}-#{@architecture}-#{nuget_package_version(project)}.nupkg"
+      end
+
+      # Nuget versioning is awesome!
+      #
+      # We can only have at most four elements in the version, all must be numeric
+      # and seperated by periods. We can't handle the version derived from the git
+      # version, which usually contains the short sha of the project. If we have
+      # four or more elements in our version string, we assume we have commits beyond
+      # the latest tag. In that case, we assume we have a version like "1.3.2.123.2g1928uf".
```

Comment:
See the line below :) I'm just assuming the first four elements are going to be numeric. The first three from the tag and the fourth the number of commits on top of the tag. The fifth element is the short sha, which is not necessarily numeric.